src/module-vsbridge-indexer-catalog/Model/ResourceModel/Product/Bundle.php

```diff
@@ -167,15 +186,43 @@ private function initSelection()
     /**
      * @return array
      */
-    private function getBundleSelections()
+    private function getBundleSelections($storeId)
     {
         $productIds = $this->getBundleIds();
+        $connection = $this->getConnection();
 
-        $select = $this->getConnection()->select()->from(
+        $select = $connection->select()->from(
             ['selection' => $this->resource->getTableName('catalog_product_bundle_selection')]
         );
+        $productIdColumn = 'parent_product_id';
+
+        if (!$this->catalogHelper->isPriceGlobal() && $storeId) {
```

Comment:
`$storeId` in our case is always present (it's never 0) so I think we can simplify condition to
```
if (!$this->catalogHelper->isPriceGlobal())
```